## Review — correct, and not over-design. One **optional** item, then merge.

Read both correction comments before the bodies, as advised. Verified in my own
worktree at `2130f153`.

### Is the OR over-design? No — it is the screen reading what the kernel ran on

I judged it against the three axes rather than by feel.

**Correctness.** The screen's job is to say the long-run variance rests on too
few lag products. The Bartlett sum forms those products on
`(n_periods, har_lags)`. Reading only `(n_periods_finite, resolved_har_lags)`
was the screen measuring a different sample than the one the kernel used — the
same defect as #1034/#1035/#1036, one layer further out. Adding the OR is not
new machinery; it is the existing screen reading the right pair.

**Style consistency.** `applied_thin` goes through
`_hac_bandwidth_ill_conditioned`, so it inherits the `n >= MIN_PERIODS_WARN`
floor. That floor is the mechanism the house convention relies on to keep the
two diagnostics non-redundant (`fm_beta`: *"below 30 periods the binding problem
is already the short score series, not a second bandwidth echo"*). The new term
does not bypass it.

**Proportionality.** The cost is bounded and you measured it. I confirmed the
premise behind the docstring correction rather than taking it — on `main` at
`c56a1e9b`:

```
n=90 h=63:  bandwidth=True  short_periods=True   -> both fire together
n=90 h=60:  bandwidth=True  short_periods=True   -> both fire together
```

So the "partition" sentence was already false before this PR, and correcting it
is not a consequence of the change. And the 562-cell case is real: `n=45, h=2`
on `main` carries `overlapping` and `short_periods` only, with 43 design rows
under a 9-lag Bartlett sum.

### The composed message, all three forms

```
both clauses (n=90 h=60):
  the resolved Bartlett bandwidth L=30 exceeds n_periods_finite / 5 on the 90
  finite pairs and the applied Bartlett bandwidth L=29 exceeds n_periods / 5 on
  the 30 rows of the augmented design, so the long-run variance rests on too few
  lag products to be stable. The augmented design admits only 29 lags, so the
  kernel ran at L=29.

applied only (n=65 h=2 lags=13):
  the applied Bartlett bandwidth L=13 exceeds n_periods / 5 on the 63 rows of
  the augmented design, so ...

resolved only (n=90 h=63):
  the resolved Bartlett bandwidth L=30 exceeds n_periods_finite / 5 on the 90
  finite pairs, so ... The augmented design admits only 26 lags, so the kernel
  ran at L=26.
```

No #1035 defect: each clause's printed count is the value under the key it
names — `n_periods_finite=90` against "the 90 finite pairs",
`n_periods=63` against "the 63 rows". Checked against the metadata in each cell.

### Optional — the `narrowed` tail is redundant exactly when the applied clause fires

You asked whether the composition reads well in both-clause cells. It does,
except at the end. In the first example above the applied clause has already
said `L=29` on 30 augmented rows, and the tail then says *"The augmented design
admits only 29 lags, so the kernel ran at L=29."* — the same two facts again.

The tail exists (from #1038) so the text cannot contradict
`metadata["har_lags"]` when only the resolved clause prints. When `applied_thin`
fires, the applied clause already prints `fit.lags_used`, which *is*
`har_lags` — so the contradiction it guards against cannot arise, and the
sentence is pure repetition. It is reachable whenever a clip coincides with
`n_used >= 30`, e.g. `n=90, h=60`.

One condition (`and not applied_thin`) closes it. **Optional** because nothing
it says is untrue — this is message quality, not correctness. Your call: push it
and I merge, or say no and I merge as-is. I will not raise it again either way.

### The pin is honest — keep it

You asked whether it reads as coverage it does not have. It states its own scope
in the docstring, and I verified that scope by mutation rather than trusting the
report:

```
floor 1.0 -> 0.25 only      1 passed
overlap cap removed only    1 passed
BOTH loosened               1 failed
```

Exactly as documented. A pin whose docstring says "this only fires if X and Y
both move" and which demonstrably only fires then is not misleading — it is
labelled correctly. It runs in 0.90s and carries `assert clipped > 1000`, so it
cannot pass by enumerating nothing. I would not delete it.

### Your requested mutation

Dropping the `applied_thin` term from the condition:

```
assert bandwidth, "the design the kernel ran on is thin and must be reported"
E   AssertionError: assert []
```

Fails at that line, not earlier — so the three cell-validity assertions above it
hold, and the cell proves what the test claims.

### Gates, re-run because of `--no-verify`

`3850 passed, 45 skipped`; `ruff check` and `ruff format --check` clean; `mypy
factrix` clean; `mkdocs build --strict` builds with no generated-file drift. CI
counted rather than filtered: **19 rollup entries, 17 pass / 2 skipping**,
`mergeStateStatus CLEAN`.

--- a/factrix/metrics/predictive_beta.py
+++ b/factrix/metrics/predictive_beta.py
@@ -459,10 +459,14 @@ def predictive_beta(
                 "of the augmented design"
             )
         # Where the kernel ran narrower than the resolved bandwidth, say so,
-        # so the text cannot contradict metadata["har_lags"].
+        # so the text cannot contradict metadata["har_lags"] (#1038). The
+        # applied clause already prints ``fit.lags_used``, which IS
+        # ``har_lags``, so when it fires the contradiction this tail guards
+        # cannot arise and the tail would only repeat two facts already in
+        # the sentence.
         narrowed = (
             ""
-            if fit.lags_used == resolved_har_lags
+            if fit.lags_used == resolved_har_lags or applied_thin
             else (
                 f" The augmented design admits only {fit.n_used - 1} lags, "
                 f"so the kernel ran at L={fit.lags_used}."

--- a/tests/stats/test_amihud_hurvich_bandwidth_report.py
+++ b/tests/stats/test_amihud_hurvich_bandwidth_report.py
@@ -211,3 +211,38 @@ def test_pinning_the_clip_moves_no_effective_degrees_of_freedom() -> None:
                     f"L={resolved} -> {used}"
                 )
     assert clipped > 1000, "the enumeration must actually reach clipped cells"
+
+
+def test_the_clip_tail_drops_where_the_applied_clause_already_said_it() -> None:
+    """#1047 review: the tail must not repeat what the applied clause printed.
+
+    The tail exists (#1038) so the text cannot contradict
+    ``metadata["har_lags"]`` when only the resolved clause prints a bandwidth.
+    Where the applied clause fires it prints ``har_lags`` itself, so the
+    contradiction cannot arise and the tail would restate the design's row
+    count and applied bandwidth a second time.
+    """
+    rng = np.random.default_rng(9060)
+    n, h = 90, 60
+    x = rng.standard_normal(n)
+    y = 0.1 * x + rng.standard_normal(n)
+
+    with warnings.catch_warnings(record=True) as caught:
+        warnings.simplefilter("always", UserWarning)
+        result = predictive_beta(_ts_panel(x, y), overlap_periods=h)
+    message = next(
+        str(item.message)
+        for item in caught
+        if "hac_bandwidth_ill_conditioned" in str(item.message)
+    )
+    metadata = result.metadata
+
+    # The cell: the bandwidth was clipped AND the applied clause fires, which
+    # is the only combination the tail is dropped for.
+    assert metadata["har_lags"] == metadata["n_periods"] - 1
+    assert (
+        f"the applied Bartlett bandwidth L={metadata['har_lags']} exceeds "
+        f"n_periods / {_MIN_PERIODS_PER_LAG} on the {metadata['n_periods']} rows "
+        "of the augmented design"
+    ) in message
+    assert "admits only" not in message

